File: LexerPLSharpVersion/LexerPLSharpVersion/constants/Brackets.cs
using LexerPLSharpVersion.token;

namespace LexerPLSharpVersion.constants {
   public static class Brackets {
      private const char CloseRoundBracket = ')';
      private const char OpenRoundBracket = '(';
      
      public static bool IsBracket(char character) {
         switch (character){
            case CloseRoundBracket:
            case OpenRoundBracket:
               return true;
            default:
               return false;
         }
      }
      
      public static TokenType DetectBracketType(char character) {
         switch (character){
            case CloseRoundBracket:
               return TokenType.CloseRoundBracket;
            case OpenRoundBracket:
               return TokenType.OpenRoundBracket;
            default:
               return TokenType.Unknown;
         }
      }
   }
}

File: OwnProgrammingLanguage/src/main/java/lexer/constants/Brackets.java
package lexer.constants;

import token.TokenType;

public class Brackets {
    private static final char OPEN_ROUND_BRACKET = '(';
    private static final char CLOSE_ROUND_BRACKET = ')';

    public static boolean IsBracket(char character) {
        switch (character){
            case OPEN_ROUND_BRACKET:
            case CLOSE_ROUND_BRACKET:
                return true;
            default:
                return false;
        }
    }

    public static TokenType detectBracketType(char character) {
        switch (character){
            case OPEN_ROUND_BRACKET:
                return TokenType.OPEN_ROUND_BRACKET;
            case CLOSE_ROUND_BRACKET:
                return TokenType.CLOSE_ROUND_BRACKET;
            default:
                return TokenType.UNKNOWN;
        }
    }
}
